spec/schemas/flow_framework.common.yaml

```diff
@@ -6,6 +6,34 @@ info:
 paths: { }
 components:
   schemas:
+    AllowDelete:
+      description: A comma-separated list of resource IDs to be deprovisioned. Required if deleting resources of type index_name or pipeline_id.
```

Comment:
Not sure if we should leave as is or make a bit more generic here.  I don't really like the specification of the specific type of resources. This specific list is correct as-of today but may be expanded and I think a change might get lost here.  Would prefer a more generic "required when deleting resources has a high potential for data loss" or similar.